PURL-TYPES.rst

```diff
@@ -65,6 +65,18 @@ cargo
       pkg:cargo/clap@2.33.0
       pkg:cargo/structopt@0.3.11
 
+cocoapods
```

Comment:
there's already an existing `cocoapods` section on line 40 above (though now that I look at it, it's not in alphabetical order)

Reply:
Ah I missed it because of the ordering and as it was listed at the bottom. Removed the one I added and replaced it with the one that already existed.